connectivity/manifests/egress-gateway-policy.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: cilium.io/v2
+kind: CiliumEgressGatewayPolicy
+metadata:
+  name: cegp-sample
+spec:
+  selectors:
+  - podSelector:
+      matchLabels:
+        io.kubernetes.pod.namespace: cilium-test
+        kind: client
+  destinationCIDRs:
+  - 0.0.0.0/0
+  egressGateway:
+    nodeSelector:
+      matchLabels:
+        kubernetes.io/hostname: NODE_NAME_PLACEHOLDER
+    interface: eth0
```

Comment:
Is this interface name available on all machines? If not, can we make it a variable? I assume this can be done similarly to [here](https://github.com/cilium/cilium/blob/main/test/k8s/manifests/egress-ip-deployment.yaml#L31:~:text=add%20%24EGRESS_IP%20dev-,%24(ip%20route%20get%201.1.1.1%20%7C%20sed%20%2Dn%20%27s/.*dev%20%5C%5C(%5B%5E%5C%5C%20%5D*%5C%5C).*/%5C%5C1/p%27),-%3B%20sleep%20inf%22)?

Reply:
Good point, I'll just get rid of `interface: eth0` altogether from the policy, then the egressgw manager will select the interface for the default route

Reply:
Tested locally:

```
🏃 Running tests...

[=] Skipping Test [north-south-loadbalancing]

[=] Skipping Test [pod-to-pod-encryption]

[=] Skipping Test [node-to-node-encryption]
[=] Test [egress-gateway]
..

✅ All 1 tests (2 actions) successful, 3 tests skipped, 0 scenarios skipped.
```